app/oxalis/security/UserAwareRequestLogging.scala

```diff
@@ -0,0 +1,33 @@
+package oxalis.security
+
+import com.mohiva.play.silhouette.api.actions.{SecuredRequest, UserAwareRequest}
+import com.scalableminds.util.requestlogging.AbstractRequestLogging
+import play.api.mvc.{Request, Result}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait UserAwareRequestLogging extends AbstractRequestLogging {
+
+  case class RequesterEmailOpt(email: Option[String]) //forcing implicit conversion
```

Comment:
So, this is needed because if we would define the parameter `emailOpt` of `log` as an `Option[String]` we could not control that the email is passed as the implicit parameter, right?